### PR TITLE
Fix console color and bump LTS baseline

### PR DESCRIPTION
**Pull request by jonesbusy**
Hi,

This PR fix console color for issue https://github.com/jenkinsci/material-theme-plugin/issues/16 and https://github.com/jenkinsci/material-theme-plugin/issues/18 and bump LTS baseline

console_fix
console_fix2

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -17,8 +17,7 @@
     <gitHubRepo>jenkinsci/material-theme-plugin</gitHubRepo>
     <revision>0.4.1</revision>
     <changelist>-SNAPSHOT</changelist>
-    <jenkins.version>2.336</jenkins.version>
-    <java.level>8</java.level>
+    <jenkins.version>2.346.1</jenkins.version>
```
Comment (timbrown5) [inline pom.xml:20]: If this a functional change? I try to keep the Jenkins version tied to the minimum supported (to allow users to use the new features when stuck on an older version).

If this is required, please bump the minor version.

Comment (jonesbusy) [reply pom.xml:20]: I think it was bumped before the LTS 2.346.* was released (due to theme manager changes)

I followed https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/ "Prefer an LTS version over weekly versions" and "try to stay on the first release in that line"

But I you prefer to stay on the weekly baseline that's fine for me

**Comment by jonesbusy** [comment]
Resolved conflicts

**Comment by jonesbusy** [comment]
Hi! 

Any updates ?

Regards,

**Comment by offa** [comment]
ping @timbrown5

**Comment by jonesbusy** [comment]
ping @timbrown5

**Comment by dbsanfte** [comment]
@timbrown5 another ping, waiting on this :)

**Review by timbrown5 — APPROVED** [review]

**Comment by offa** [comment]
@timbrown5 could you consider a new release containing this fix?

**Comment by jonesbusy** [comment]
+1 for a release. Thanks!

**Pull request merged**